README.md

```diff
@@ -44,6 +44,8 @@ npm install nuxt-google-optimize --save
     // experimentsDir: '~/experiments',
     // maxAge: 60 * 60 * 24 * 7 // 1 Week
     // pushPlugin: true,
+    // eventHandler: 'ga', // 'ga' = Google Analytics integration || 'gtm' = @nuxtjs/gtm-module integration || 'dataLayer' = GTM integration
+    // dataLayer: 'dataLayer', // eventHandler option has to be 'dataLayer' when implementing custom GTM dataLayer
```

Comment:
Should this be left empty when using default ga?

Reply:
This does show the possible options with set defaults for clarity. At least for me, it is easier to understand what values are set when not specifically changed. This could be documented in some other way as well but this is the first that came to my mind.

I did change the order in the handler method check. For now, it defaults to "ga" when an unsupported string is filled in the eventHandler or the setting is not set.